src/targets/gpu/jit/mlir.cpp

```diff
@@ -198,8 +200,14 @@ struct mlir_compiler : compiler<mlir_compiler>
                                               const operation&,
                                               bool exhaustive) const
     {
+        static const auto mxr_loc = string_value_of(MIGRAPHX_MLIR_DUMP_TO_MXR{});
```

Comment:
add to ENV var document. 

Also what's the usefulness of having MXRs for each MLIR module ?

Reply:
Added docs.